ulib/ml/FStar_UInt8.ml

```diff
@@ -74,7 +74,8 @@ let op_Less_Equals_Hat = lte
 
 let of_string s = int_of_string s
 let to_string s = string_of_int s
-let to_string_hex s = Printf.sprintf "%02x" s
+let to_string_hex s = Printf.sprintf "0x%x" s
```

Comment:
@mtzguido Was the 0x prefix intended here and not on the function below?

Reply:
Yes, the criterion is that `to_string_hex` prints with a leading `0x`, without any fixed width check. OTOH, `to_string_hex_pad` is fixed width, without a `0x`. It's up for discussion of course, but it's consistent with all the other current implementations.